tech-far-hub/src/url-utils.tsx

```diff
@@ -0,0 +1,59 @@
+import { IURLInfo, DownloadFileType } from "./types";
+
+export class URLInfo implements IURLInfo {
```

Comment:
class components!

Reply:
I was so happy to just write like a normal class tbh